source/jormungandr/tests/routing_tests_new_default.py

```diff
@@ -405,3 +405,11 @@ class TestNewDefaultOneDeadRegion(OneDeadRegion, NewDefaultScenarioAbstractTestF
 @config({"scenario": "new_default"})
 class TestNewDefaultWithoutPt(WithoutPt, NewDefaultScenarioAbstractTestFixture):
     pass
+
+@config({"scenario": "new_default"})
+class TestNewDefaultMinNbJourneys(JourneysMinNbJourneys, NewDefaultScenarioAbstractTestFixture):
+    pass
+
+@config({"scenario": "new_default"})
+class TestNewDefaultWithNightBusFilter(JourneysWithNightBusFilter, NewDefaultScenarioAbstractTestFixture):
+    pass
```

Comment:
Are these two test classes **JourneysMinNbJourneys** and **JourneysWithNightBusFilter** not supported in new_default and distributed or deactivated temporarily?

Reply:
I have created two differents classes to segment tests, *JourneysMinNbJourneys* and *JourneysWithNightBusFilter*. 
They are called in **new_default** and **distributed**.